python/src/equistore/block.py

```diff
@@ -224,6 +270,34 @@ def add_gradient(
         """
         Add a set of gradients with respect to ``parameters`` in this block.
 
+        Here is an example of how to use this method:
```

Comment:
Same comments as for the last example apply here. Can you swap the examples between `gradient` and  `add_gradient`? At the moment the examples in `gradient` are more comprehensive, but I believe that a user will rather run into problems when using `add_gradient` and try to read the doc from this function. So it would be better if this one is more comprehensive.